lib/spack/spack/cmd/create.py

```diff
@@ -268,6 +268,38 @@ def __init__(self, name, *args):
         super(RPackageTemplate, self).__init__(name, *args)
 
 
+class PerlPackageTemplate(PackageTemplate):
+    """Provides appropriate overrides for Perl extensions"""
+    base_class_name = 'PerlPackage'
+
+    dependencies = """\
+    # FIXME: Add dependencies if required.
+    # depends_on('perl-foo')
+    # depends_on('barbaz', type=('build', 'link', 'run'))
+    """
+
+    body = """\
+    # FIXME: Select appropriate perl build method:
+    # build_method = 'Makefile.PL' # Default
+    # build_method = 'Build.PL'    # Needed by some packages
```

Comment:
Do any packages provide both? And if so, is there any advantage to using one or the other? If not, let's just see which file exists and use that. No need to require the user to specify this.